[2023-08-18T23:10:43.936+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=dags_bash_select_dog/run_id=manual__2023-08-18T23:10:41.081828+00:00/task_id=t1_dachshund permission to 509
[2023-08-18T23:10:43.938+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=dags_bash_select_dog/run_id=manual__2023-08-18T23:10:41.081828+00:00/task_id=t1_dachshund permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dags_bash_select_dog/run_id=manual__2023-08-18T23:10:41.081828+00:00/task_id=t1_dachshund'
[2023-08-18T23:10:43.984+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=dags_bash_select_dog/run_id=manual__2023-08-18T23:10:41.081828+00:00/task_id=t1_dachshund permission to 509
[2023-08-18T23:10:43.986+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=dags_bash_select_dog/run_id=manual__2023-08-18T23:10:41.081828+00:00/task_id=t1_dachshund permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dags_bash_select_dog/run_id=manual__2023-08-18T23:10:41.081828+00:00/task_id=t1_dachshund'
[2023-08-18T23:10:44.166+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dags_bash_select_dog.t1_dachshund manual__2023-08-18T23:10:41.081828+00:00 [queued]>
[2023-08-18T23:10:44.200+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dags_bash_select_dog.t1_dachshund manual__2023-08-18T23:10:41.081828+00:00 [queued]>
[2023-08-18T23:10:44.202+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-08-18T23:10:44.284+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): t1_dachshund> on 2023-08-18 23:10:41.081828+00:00
[2023-08-18T23:10:44.299+0000] {standard_task_runner.py:57} INFO - Started process 358 to run task
[2023-08-18T23:10:44.315+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dags_bash_select_dog', 't1_dachshund', 'manual__2023-08-18T23:10:41.081828+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/dags_bash_select_dog.py', '--cfg-path', '/tmp/tmp0un64_1t']
[2023-08-18T23:10:44.324+0000] {standard_task_runner.py:85} INFO - Job 14: Subtask t1_dachshund
[2023-08-18T23:10:44.529+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=dags_bash_select_dog/run_id=manual__2023-08-18T23:10:41.081828+00:00/task_id=t1_dachshund permission to 509
[2023-08-18T23:10:44.531+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=dags_bash_select_dog/run_id=manual__2023-08-18T23:10:41.081828+00:00/task_id=t1_dachshund permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dags_bash_select_dog/run_id=manual__2023-08-18T23:10:41.081828+00:00/task_id=t1_dachshund'
[2023-08-18T23:10:44.540+0000] {task_command.py:410} INFO - Running <TaskInstance: dags_bash_select_dog.t1_dachshund manual__2023-08-18T23:10:41.081828+00:00 [running]> on host 7ff4ca19a8ff
[2023-08-18T23:10:44.907+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dags_bash_select_dog' AIRFLOW_CTX_TASK_ID='t1_dachshund' AIRFLOW_CTX_EXECUTION_DATE='2023-08-18T23:10:41.081828+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-18T23:10:41.081828+00:00'
[2023-08-18T23:10:44.916+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-18T23:10:44.920+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '/opt/***/plugins/shell/select_dog.sh DACHSHUND']
[2023-08-18T23:10:44.962+0000] {subprocess.py:86} INFO - Output:
[2023-08-18T23:10:45.006+0000] {subprocess.py:93} INFO - You selected DACHSHUND!
[2023-08-18T23:10:45.012+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-18T23:10:45.128+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=dags_bash_select_dog, task_id=t1_dachshund, execution_date=20230818T231041, start_date=20230818T231044, end_date=20230818T231045
[2023-08-18T23:10:45.249+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-08-18T23:10:45.286+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
